forc-plugins/forc-crypto/src/address.rs

```diff
@@ -0,0 +1,30 @@
+use anyhow::anyhow;
+use fuel_crypto::fuel_types::Address;
+use fuels_core::types::bech32::Bech32Address;
+use std::str::FromStr;
+
+/// Takes a valid address in any supported format and returns them in all
+/// supported format. This is meant to be a tool that can be used to convert any
+/// address format to all other formats
+pub fn dump_address<T: AsRef<[u8]>>(data: T) -> anyhow::Result<String> {
```

Comment:
This looks much readable so +1 here with Josh. I feel like this also increases reusability, I can see `handle_string_conversion` reused somewhere later on.

Also looks like you don't need to go to `String` for the purpose of converting it to a `Bech32Address` or `Address`. You seem like in need of a `&str` since they use `from_str()` and thus you can use [`str::from_utf8()`](https://doc.rust-lang.org/std/str/fn.from_utf8.html) which does not allocate, but uses `mem::transmute` [under the hood](https://doc.rust-lang.org/src/core/str/converts.rs.html#173).

Reply:
Done in a837b9c59d